contrib/raspberrypi3/install_electrumx.sh

```
@@ -24,4 +24,4 @@ sudo pip3 install plyvel
 # install electrumx
 git clone https://github.com/spesmilo/electrumx.git
 cd electrumx
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)